i3lock.c

```diff
@@ -658,8 +659,18 @@ static void raise_loop(xcb_window_t window) {
     }
 }
 
+char* whoami(void) {
+    uid_t uid = geteuid();
```

Comment:
See http://docstore.mik.ua/orelly/networking/puis/ch04_03.htm — I think we should use `getuid()` instead of `geteuid()` here.